package.json

```diff
@@ -26,6 +26,7 @@
     "build": "microbundle --target node --compress --sourcemap false",
     "clean": "node -e \"fs.rmSync('./dist', { recursive: true, force: true });\"",
     "dev": "microbundle watch",
+    "pretest": "npm run build",
```

Comment:
Added this to make it easier to test. I'm assuming you run with `watch` while developing, but for a newbie to the repo, it was confusing to have tests fail out of the box.

Reply:
Great point, I was fixing part of this yesterday with actions on GH and instead of fixing it at the package.json level, I did it at the action level! Thanks for catching this!